CMakeLists.txt

```diff
@@ -1,25 +1,19 @@
 
 cmake_minimum_required (VERSION 3.6)
-project (sleepydiscord)
-
-# Not reccomended but I do it anyway
-file(GLOB_RECURSE SOURCE_FILES "${CMAKE_SOURCE_DIR}/sleepy_discord/*.c*")
+project(sleepydiscord)
 
 set(CCOMMON_FLAGS "-mtune=generic -O3 -Wall -c")
 
 set(CMAKE_CXX_FLAGS_DEBUG "${CCOMMON_FLAGS} ${CMAKE_CXX_FLAGS_DEBUG} -g3 -ggdb")
 set(CMAKE_CXX_FLAGS_RELEASE "${CCOMMON_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE}")
-set(CMAKE_CC_FLAGS_DEBUG "${CCOMMON_FLAGS} ${CMAKE_CXX_FLAGS_DEBUG} -g3 -ggdb")
-set(CMAKE_CC_FLAGS_RELEASE "${CCOMMON_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE}")
-
-include_directories("${CMAKE_SOURCE_DIR}/include/sleepy_discord")
-include_directories("${CMAKE_SOURCE_DIR}/include/sleepy_discord/IncludeNonexistent")
-message(STATUS "${INCLUDE_DIRECTORIES}")
 
 # Open SSL
 find_package(OpenSSL REQUIRED)
 
+# Not reccomended but I do it anyway
+file(GLOB_RECURSE SOURCE_FILES "${CMAKE_SOURCE_DIR}/sleepy_discord/*.c*")
 # why static >_>
 add_library(sleepydiscord STATIC "${SOURCE_FILES}")
 
+target_include_directories(sleepydiscord PUBLIC "include/sleepy_discord/" "include/sleepy_discord/IncludeNonexistent/")
```

Comment:
I think the include directories should be
```
include
deps
deps/include
include/sleepy_discord/IncludeNonexistent```

Reply:
shouldnt deps and deps/include be done by the user when using an already compiled sleepy-discord library?
Also, i dont see the folders for deps anywhere.

Reply:
well it's created by the user or the setup script. It's used for a place to have all the library the user will use with sleepy discord so maybe it shouldn't be there.